src/LibYear/Output/JsonOutput.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Text.Json;
+using LibYear.Core;
+using Spectre.Console;
+
+namespace LibYear.Output;
+
+internal sealed class JsonOutput : IOutput
```

Comment:
Let's stick all the JSON-related output in a `LibYear.Output.JSON` namespace (and same w/table) so we can keep the implementation details separate from the common logic